wikimedia/wikipedia-ios
# Talk pages - fix navigating to archived talk page

**Phabricator:** https://phabricator.wikimedia.org/T318610

### Notes
* This PR adds encoding to talk page titles to fix a redirect error from talk page archives

### Test Steps
1. Go to a talk page coffee roll, click on an archive page
2. On the archive page, click on any link. 
3. The clicked link should open normally
